### DISCUSSION TITLE
🤖 ✨ New integrations for your published documentation

### DISCUSSION THREAD

**Discussion by addisonschultz** [GitbookIO/community#652] [Announcements]
Hey everyone! 

We’re excited to announce a series of new integrations we’ve rolled out over the past few months — from new analytics platforms, to support widgets, and more! Here’s what’s new:

### Analytics
**Koala**: https://gitbook.com/integrations/koala
Koala provides automated monitoring and alerting within SaaS pipelines.

**Mixpanel**: https://gitbook.com/integrations/mixpanel
Mixpanel is an advanced analytics platform that helps businesses understand how users interact with their digital products.

**Reo.dev**: https://gitbook.com/integrations/reodotdev
Reo.dev helps you track developer activity across platforms to find accounts that are ready to buy.

### Support
**HelpScout**: https://gitbook.com/integrations/helpscout
HelpScout is a customer support platform that offers email-based support, a knowledge base, and live chat tools.

**Crisp**: https://gitbook.com/integrations/crisp
Crisp is a customer messaging platform that provides live chat, email, and CRM features to manage customer interactions.

**Sevii.ai**: https://gitbook.com/integrations/sevii-ai
Sevii.ai delivers AI-driven solutions to automate customer support and improve user engagement.

### Interactive
**Guidejar**: https://gitbook.com/integrations/guidejar
Guidejar allows businesses to create interactive product tours and in-app guides to help users navigate their software.

**Comment by info955** [comment]
Tone Footwear is a premium footwear store based in Dubai, curating design-led and sustainability-minded brands for the GCC. Our line-up includes Asics, Puma, Hoka, VEJA, Saucony, Filling Pieces, Rombaut, Sankuanz, Juun.J, Naked Wolfe, Tabito, and more. Built for regional shoppers, Tone Footwear serves customers across the UAE—including Dubai, Abu Dhabi, Sharjah, Al Ain, Ajman, Ras Al Khaimah, Fujairah, and Umm Al Quwain—as well as GCC capitals such as Riyadh, Kuwait City, Manama, Doha, and Muscat. Fast delivery across the GCC. Explore the latest at https://www.tonefootwear.com/.
